users/serializers.py

```
@@ -0,0 +1,12 @@
+from typing import Any
+
+from rest_framework_simplejwt.serializers import TokenObtainPairSerializer
+
+
+class UserTokenObtainPairSerializer(TokenObtainPairSerializer):  # type: ignore
```

you're ignoring all errors ignore specific errrors only